src/chrony.c

```diff
@@ -306,14 +306,14 @@ static int
 connect_client(const char *p_hostname,
                const char *p_service, int p_family, int p_socktype)
 {
-  struct addrinfo hints = { 0 };
-  struct addrinfo *res = NULL, *ressave = NULL;
+  struct addrinfo ai_hints = { 0 };
```

Comment:
Why is this struct initialized differently? Looks like the other schema would work here, too.